### PR TITLE
update package management doc

### PR DESCRIPTION
**Pull request by zfengms**

### CODE REVIEW

File: docs/20-package-management.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ The doAzureParallel package allows you to install packages to your pool in two w
 - Installing on pool creation
 - Installing per-*foreach* loop
 
-Packages installed at the pool level benefit from only needing to be installed once per node. Each iteration of the foreach can load the library without needing to install them again. Packages installed in the foreach benefit from specifying any specific dependencies required only for that instance of the loop.
+Packages installed at the pool level benefit from only needing to be installed once per node. Each iteration of the foreach can load the library without needing to install them again. Packages installed in the foreach benefit from specifying any dependencies required only for that instance of the loop.
```
Comment (brnleehng) [inline docs/20-package-management.md]: Is this the same PR as the other one?

File: samples/package_management/bioconductor.r [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#Please see documentation at docs/20-package-management.md for more details on packagement management.
+
 # install packages
 library(devtools)
 install_github("azure/doazureparallel")
```
Comment (brnleehng) [inline samples/package_management/bioconductor.r]: We should remove these installation?

File: docs/20-package-management.md (hunk 1 above)
Comment (zfengms) [reply docs/20-package-management.md]: Yes

File: samples/package_management/bioconductor.r (hunk 2 above)
Comment (zfengms) [reply samples/package_management/bioconductor.r]: Removed.

**Pull request merged**